core/src/main/antlr4/io/delta/sql/parser/DeltaSqlBase.g4

```diff
@@ -87,8 +87,11 @@ statement
     | ALTER TABLE table=qualifiedName
         DROP CONSTRAINT (IF EXISTS)? name=identifier                    #dropTableConstraint
     | OPTIMIZE (path=STRING | table=qualifiedName)
-        (WHERE partitionPredicate = predicateToken)?
+        (WHERE partitionPredicate=predicateToken)?
```

Comment:
Is this change intended?

Reply:
Yes, because I see other rules do not have spaces around `=`.